selectors-4/Overview.bs

```diff
@@ -831,8 +829,14 @@ Pseudo-classing Pseudo-elements</h4>
 		whereas ''::first-line:hover'' only matches if the first line itself is hovered.
 	</div>
 
+	The [=logical combination pseudo-classes=]
+	'':not()'', '':is()'', and '':where()''
+	are also valid immediately following a [=pseudo-element=]
+	provided their arguments all adhere to the above restriction.
```

Comment:
Does this mean that `::before:is(:hover, :first-child)` is completely invalid, or becomes `::before:is(:hover)`?

Reply:
Seems like the latter would be more consistent with standard behavior, so I'll update the PR.